### DISCUSSION TITLE
courses/beginners-web-handbook/messenger-project-1/07-deploy

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#48] [Q&A] [answered]
# courses/beginners-web-handbook/messenger-project-1/07-deploy

바닐라코딩

https://www.vcdr.club/courses/beginners-web-handbook/messenger-project-1/07-deploy

**Comment by suyoung247** [comment, marked as answer]
동일하게 진행했음에도 오류가 발생하여 문의드립니다.
7. 업로드한 파일 커밋하기 후 홈페이지들 접속하면(ongoing.github.io)에 접속하면 아래와 같은 오류가 발생합니다
// 404
There isn't a GitHub Pages site here.

If you're trying to publish one, read the full documentation to learn how to set up GitHub Pages for your repository, organization, or user account.
//

혹시 누락한 요소가 있을까 싶어서 처음부터 다시 만들었음에도 동일한 증상이 발생합니다. 해결할 수 있는 방향을 제시해주시면 감사드리겠습니다.

**Reply by suyoung247** [reply]
깃허브 저장소 내 Security탭 -> Vulnerability alerts 탭 -> Secret scanning alerts 1건 식별되었는데 이것과 관련이 있는지 궁금합니다.

**Reply by Ken123777** [reply]
안녕하세요. 사용자 이름이 `ongoig`으로 보여지는데, 레포지토리 이름이 `ongoing.github.io`로 되어 있는것 같습니다. `ongoing`을 `ongoig`으로 수정하시고 다시 시도해보시겠어요?
